README.md

```diff
@@ -0,0 +1,73 @@
+# Pydarn 
+[![Open Source Love](https://badges.frapsoft.com/os/mit/mit.svg?v=102)](https://github.com/ellerbrock/open-source-badge/) [![Python 3.6](https://img.shields.io/badge/python-3.6-blue.svg)](https://www.python.org/downloads/release/python-360/) [![GitHub version](https://badge.fury.io/gh/boennemann%2Fbadges.svg)](http://badge.fury.io/gh/boennemann%2Fbadges)
+
+SuperDARN data visualization library. 
+
+## Getting Started 
+
+The following instructions will allow you to install and give some examples on how to use pydarn. 
+
+### Prerequisites
+
+**python 3.6+**
+
+| Ubuntu      | OpenSuse       |
+| ----------- | -------------- |
+| libyaml-dev | python3-PyYAML |
+
+You can check your python version with  
+`$ python --version` or 
+`$ python3 --version`
+### Installing 
+
+1. Clone git repository:   
+   `git clone https://github.com/SuperDARN/pydarn.git`
+
+2. Installing pydarn  
+    1. **Recommended**: Installing a [virtual environment](https://packaging.python.org/guides/installing-using-pip-and-virtualenv/), this option allows the library to install needed version of libraries without affecting system libraries.  
+        * First install the environment:  
+      `$ python3 -m pip install --user virtualenv`  
+      `$ python3 -m virtualenv <environment name>`  
+		  `$ source <environment name>/bin/activate`
+        * Navigate to where you cloned pydarn:  
+		  `$ python3 setup.py install`
+    2. Install in the system (root privileges required):  
+		   `$ sudo python3 setup.py install`
+
+### Example 
+
+#### Reading DMAP file 
+DMAP files are a SuperDARN 
+
+```python
+import pydarn
+dmap_file = "./20180410.C0.sas.fitacf"
+damp_reader = pydarn.DmapRead(dmap_file)
+dmap_data = dmap_reader.read_records() 
+
+# dmap_data[record number][paramter name].value
+print(dmap_data[0]['beam'].value) 
+```
+
+#### Turn on debugging 
+
+```python
+import pydarn
+import logging
+
+pydarn_logger = logging.getLogger('pydarn').SetLevel(logging.DEBUG)
```

Comment:
Typo: "SetLevel" should be "**s**etLevel".

Really, why doesn't logging module use underscores for methods...

Reply:
Most python libraries are actually like that. If `setLevel` is it's own inherent class to logging, it follows the PEP 8 style. However, I think it is more of the downfall of python's beastly-ness. If logging was implemented fairly early on (earliest I found was python 2.6) and PEP 8 wasn't enforced until later like 2.7 then the developers of that library probably just stuck with the old syntax for consistency and to prevent other libraries from breaking from them changing method names.  ¯\_(ツ)_/¯
  

Fixed the typo